Util/Xamarin.Build.Download/source/Xamarin.Build.Download/XamarinBuildGetArchivesToDownload.cs

```diff
@@ -57,8 +57,8 @@ public override bool Execute ()
 					taskItem.SetMetadata ("RangeStart", partialZipDownload.RangeStart.ToString ());
 					taskItem.SetMetadata ("RangeEnd", partialZipDownload.RangeEnd.ToString ());
 
-					if (!string.IsNullOrEmpty (partialZipDownload.Md5))
-						taskItem.SetMetadata ("Md5", partialZipDownload.Md5);
+					if (!string.IsNullOrEmpty (partialZipDownload.Sha256))
+						taskItem.SetMetadata ("Sha256", partialZipDownload.Sha256);
```

Comment:
I see all these nice changes, but are we backwards compatible?

Reply:
It should just skip over verifying the old md5 hashes since it no longer checks for them and no longer sets the task item, it will just never be checked to be validated.